src/components/modal/modal.jsx

```diff
@@ -0,0 +1,34 @@
+const React = require('react');
+const ReactModal = require('react-modal');
+
+const styles = require('./modal.css');
+
+class ModalComponent extends React.Component {
+    render () {
+        return (
+            <ReactModal
+                className={styles.modalContent}
+                isOpen={this.props.visible}
+                overlayClassName={styles.modalOverlay}
+                ref={m => (this.modal = m)}
+                onRequestClose={this.props.onRequestClose}
+            >
+                <div
+                    className={styles.modalCloseButton}
+                    onClick={this.props.onRequestClose}
+                >
+                    {'x'}
```

Comment:
Leaving this as-is for now, when I changed it I saw one other reason, which is that it's really easy to miss the stray `x` on that line. It's much more visible this way.